tox.ini

```diff
@@ -6,7 +6,6 @@ usedevelop = True
 whitelist_externals=rm
 setenv =
     PYTHONPATH = {toxinidir}
-    KOLIBRI_HOME = {toxinidir}/kolibrihome_test
```

Comment:
...actually `DJANGO_SETTINGS_MODULE = kolibri.deployment.default.settings.test` should have activated this, but previous Travis error indicates that it's being run with the default `~/.kolibri` home: https://travis-ci.org/learningequality/kolibri/jobs/242523615#L1649